src/Headless.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace HeadlessLaravel;
+
+use Composer\Autoload\ClassMapGenerator;
+use HeadlessLaravel\Cards\Card;
+use HeadlessLaravel\Cards\CardGroup;
+use HeadlessLaravel\Formations\Formation;
+use Illuminate\Support\Facades\Route;
+
+class Headless
+{
+    public function formations()
+    {
+        $formations = [];
+        $formationsPath = config('headless-laravel.formations_path');
+
+        if (!file_exists($formationsPath)) {
+            return $formations;
+        }
+
+        $formations = $this->getClasses($formationsPath);
+
+        foreach ($formations as $formationName) {
+            /** @var Formation $formation */
+            $formation = app($formationName);
+            Route::formation($formationName)
+                ->resource((string) $formation->guessResourceName()); // guessResourceName needs a PR to formation repo
+        }
+    }
+
+    public function cards()
```

Comment:
I would like `Headless::cards()` to return a list of classes

Change this to `routeCards()`